block-layer/codecs/dag-pb.md

```diff
@@ -0,0 +1,59 @@
+# DagPB Spec
+
+DagPB does not support the full ["IPLD Data Model."](../../data-model-layer/data-model.md)
+
+## Format
+
+The DagPB IPLD format is a legacy format implemented with a single protobuf.
+
+```protobuf
+// An IPFS MerkleDAG Link
+message PBLink {
+
+  // binary CID (with no multibase prefix) of the target object
+  optional bytes Hash = 1;
+
+  // UTF-8 string name.
+  optional string Name = 2;
+
+  // cumulative size of target object
+  optional uint64 Tsize = 3;
+}
+
+// An IPFS MerkleDAG Node
+message PBNode {
+
+  // refs to other objects
+  repeated PBLink Links = 2;
+
+  // opaque user data
+  optional bytes Data = 1;
+}
+```
+
+The objects link names are specified in the 'Name' field of the PBLink object.
+All link names in an object must either be blank or unique within the object.
+
+## Pathing
+
+There is some overlap between the Go and JavaScript implementation of DagPB. Both support pathing with link names: `/<name1>/<name2>/…`.
```

Comment:
Can we note (agree) that this is a bug?

Reply:
Probably worth linking to https://github.com/ipld/specs/issues/55 as a "discussion of this issue and how it might be resolved".

Reply:
@Stebalien We can totally agree on that. Let's discuss it at #55 (and I'll add a link to it in here).